.gitignore

```diff
@@ -75,5 +75,6 @@ terraform.rc
 /graveler/committed/mock/
 /graveler/sstable/mock/
 /pyramid/mock/*
+/onboard/mock/*
```

Comment:
can you remove the '*' from both lines - the directory is optional

Reply:
Done.